docs/modules/ROOT/pages/reference/quality-usability.adoc

```diff
@@ -0,0 +1,93 @@
+= Usability
+
+== Every Service in the Catalog Has a Structured Product Description
+
+Source::
+User
+
+Stimulus::
+Reading an AppCatalog Service documentation
+
+Environment::
+AppCatalog enabled Kubernetes cluster
+
+Artifact::
+https://kb.vshn.ch[VSHN Documentation]
+
+Response::
+The product description is available.
+
+Response measure::
+The service has a product description based on a template that covers all important aspects of the service.
+
+Rationale::
+Having product descriptions clearly communicates what is to be expected of each service.
+
+== Configuration of Services Are Done via CRs or via Service Broker
+
+Source::
+User
+
+Stimulus::
+Creates an AppCatalog Service
+
+Environment::
+AppCatalog enabled Kubernetes cluster
+
+Artifact::
+Kubectl or Service Broker API
+
+Response::
+The service is created
+
+Response measure::
+The service belongs to the same organization where the CR is applied.
+
+Rationale::
+To give users a native k8s experience. The services should be created by well-defined and simple CRs.
+
+== Connection Details Are Provided via Kubernetes Secrets or via Service Broker
+
+Source::
+User
+
+Stimulus::
+Creates an AppCatalog Service
+
+Environment::
+AppCatalog enabled Kubernetes cluster
+
+Artifact::
+Kubectl or Service Broker API
+
+Response::
+The connection details are available to the user
+
+Response measure::
+The connection details are available to the user in the form of a k8s secret.
+
+Rationale::
+To give users a native k8s experience. The secrets can be referenced from other applications and thus be easily integrated.
+
+== All Service Logs Are Stored Centrally
+
+Source::
+User
+
+Stimulus::
+Opens the Kubernetes cluster logging solution
+
+Environment::
+AppCatalog enabled Kubernetes cluster
+
+Artifact::
+Webbrowser
+
+Response::
+All logs of the service are available
+
+Response measure::
+The service's logs are available up to the normal logging retention.
+
+Rationale::
+To give the users more flexibility with debugging issues and using the services, we will provide access to the service's logs.
```

Comment:
I doubt we can make this part of the first iteration.
I would exclude it for the time being, unless it's already defined and I missed it?

Reply:
Yes, but there's a lot of open questions how we can grant access to the namespaces and their logs without giving potentially destructive access to service namespaces that are more or less treated as blackboxes for users.

Besides, this could be fine with APPUiO Cloud, but having this requirement here basically means that it applies also to each existing and future private managed cluster, where definitely not all clusters have even central logging enabled. So I would remove it for the time being as a general requirement, and list this feature as exclusive for APPUiO Cloud for now. At least until we know if and how we aim to achieve the same for private clusters. 

The difference is that with a clear dedicated requirement like this we commit ourselves that each AppCat environment fulfills it, even in cases where OpenShift logging isn't available and we would have to engineer something ourselves. However, if the central logging feature is outsourced to the platform (OpenShift), then we can argue that this feature availability depends on the platform.

Reply:
We should rephrase the requirement to something like "User Has Access To Logs of Service", which doesn't mention how and if these logs are stored for a long time. Technically speaking an "oc -n my-black-box-service logs mydb" should be possible.

From a users' perspective having access to logs is a USP, therefore we should keep this requirement. If we see over time that this isn't feasible, we can always revisit this requirement and adapt.

Reply:
I see @ccremer's point.

My suggestion, I rephrase it for the time being, that this only applies to the environment "APPUiO Cloud". We can re-visit it in a later stage ans extend as needed.